lang/python/django-compressor/Makefile

```diff
@@ -9,38 +9,68 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=django-compressor
 PKG_VERSION:=2.2
```

Comment:
The latest one is 2.3.

Reply:
ack;
but i would defer updates of packages;
mostly due to the fact that it's some extra unknowns

most packages in the seafile dependency chain require updating;
i will do an update later, when also updating seafile